opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -146,7 +148,10 @@ impl Drop for LoggerProviderInner {
     fn drop(&mut self) {
         for processor in &mut self.processors {
             if let Err(err) = processor.shutdown() {
-                global::handle_error(err);
+                otel_warn!(
+                    name: "LoggerProvider.Drop.AlreadyShutdown",
```

Comment:
not sure how we assume that the error is due to "AlreadyShutdown".

Reply:
Yes good catch I have changed it to `LoggerProvider.Drop.ShutdownError`. Have kept it as warning, as the error is during shutdown of LoggerProvider, and should be less destructive ( I think)